src/Cake.Common/IO/DirectoryAliases.cs

```diff
@@ -144,6 +144,30 @@ public static void CleanDirectories(this ICakeContext context, string pattern)
         }
 
         /// <summary>
+        /// Cleans the directories matching the specified pattern.
+        /// Cleaning the directory will remove all it's content but not the directory itself.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <param name="pattern">The pattern to match.</param>
+        /// <param name="predicate">The predicate used to filter directories based on file system information.</param>
+        /// <example>
```

Comment:
Ok will go thru and fix that and all existing example inconsistencies in that class.